assets/figures/yap_system.tex

```diff
@@ -0,0 +1,60 @@
+
+
+
+\begin{tikzpicture}[scale = 0.67]
+
+\definecolor {dgray}{rgb}{122,122,122}
+\definecolor {lgray}{rgb}{194,194,194}
+
+% Payee
+	\node 			(payee) at (12,7) {\includegraphics[width = 1 cm]{../assets/images/avatar.png}};
+
+% Payer
+	\node 			(payer) at (0,7) {\includegraphics[width = 1 cm]{../assets/images/avatar.png}};
+	
+
+% Stone
+	\node (outer) at (10.5,1.5) [circle, minimum width=2.5cm, draw = gray, fill = lightgray]{};
+
+	\node (inner) at (10.5,1.5) [circle, minimum width=0.9cm, draw = gray, fill = white]{};
```

Comment:
We should use the same yap stones figure in all slide decks. Suggestion in commit 0221116. Suggestion after merging main into the branch:

```suggestion:
% Stone
	\node[minimum size = 0.8cm] (stone) at (10.5,1.5) {\input{../assets/figures/yap.tex}};
```

Reply:
Suggestion adopted with commit e5f1d7c